cdap-common/src/main/java/io/cdap/cdap/common/metrics/MetricsReporterHook.java

```diff
@@ -86,6 +89,9 @@ public boolean preCall(HttpRequest request, HttpResponder responder, HandlerInfo
 
   @Override
   public void postCall(HttpRequest request, HttpResponseStatus status, HandlerInfo handlerInfo) {
+
+    setAuditLogInfoInContext(request, status, handlerInfo);
```

Comment:
I would either have it as a separate hook or move this hook out of metrics as this has nothing to do with metrics